openfga/v1/authzmodel.proto

```diff
@@ -200,6 +219,19 @@ message Condition {
     (validate.rules).map.max_pairs = 25,
     (validate.rules).map.keys.string = {pattern: "^[^:#@\\s]{1,50}$"}
   ];
+
+  ConditionMetadata metadata = 4;
+}
+
+message ConditionMetadata {
```

Comment:
```suggestion
message ConditionSourceMetadata {
```

Reply:
I think that it makes sense to separate this yeah, do you see there being an issue with having a difference in the structure between conditions and type defintions/relations? Not from a technical perspective but from a readability/usability perspective?

Reply:
Trailer metadata for module and source info stuff makes sense and is consistent with the usage patterns above. It also separates the core fields from general meta stuff. Looks good to me 👍